encoding/form/proto_encode.go

```diff
@@ -137,14 +137,12 @@ func EncodeField(fieldDescriptor protoreflect.FieldDescriptor, value protoreflec
 		}
 		desc := fieldDescriptor.Enum().Values().ByNumber(value.Enum())
 		return string(desc.Name()), nil
-	case protoreflect.StringKind:
-		return value.String(), nil
 	case protoreflect.BytesKind:
 		return base64.URLEncoding.EncodeToString(value.Bytes()), nil
 	case protoreflect.MessageKind, protoreflect.GroupKind:
 		return encodeMessage(fieldDescriptor.Message(), value)
 	default:
-		return fmt.Sprint(value.Interface()), nil
+		return value.String(), nil
```

Comment:
Should this be a feature or fix?

Reply:
![image](https://github.com/go-kratos/kratos/assets/27986239/1b2e8a0d-1aaa-4dae-9d7f-48e4a2534a71)

`value.String()` already contains `fmt.Sprint(value.Interface())`, for non-special types, just use `.String()`, optimized by the way.

Reply:
What I'm talking about is the title issue

Reply:
done @shenqidebaozi